pkg/render/manager.go

```diff
@@ -578,12 +578,6 @@ func managerClusterRole(impersonationOnly bool) *rbacv1.ClusterRole {
 
 	if !impersonationOnly {
 		cr.Rules = append(cr.Rules,
-			rbacv1.PolicyRule{
-
-				APIGroups: []string{"authentication.k8s.io"},
-				Resources: []string{"tokenreviews"},
-				Verbs:     []string{"create"},
-			},
```

Comment:
Does Voltron not need at least one of `tokenreviews` or `authenticationreviews` for its AuthN checking?

Reply:
No, the Authorization headers that it receives are sent to /apis/projectcalico.org/authenticationreviews and for this reason, we can remove the permission from here, but we need to add it to the sa's that do cross cluster requests.

Reply:
There is a thread on this PR with Karthik that further clarifies the flow ^